### PR TITLE
Initial responsive changes + dashboard

### PR DESCRIPTION
**Pull request by benwerd**
There's more to do (including locks and adding the Unlock logo to the header).

This PR uses @cellog's media idea. Thanks @cellog!

Here's the header of the dashboard:

screen shot 2018-12-13 at 6 20 45 pm

Here's the homepage, which remains unchanged:

screen shot 2018-12-13 at 6 20 22 pm

Here's the dashboard in desktop view, which is also unchanged:

screen shot 2018-12-13 at 6 21 02 pm

### CODE REVIEW

File: unlock-app/src/components/Components.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components'
 
 export const Section = styled.section.attrs(props => ({
-  'id': props.anchor,
+  id: props.anchor,
```
[the remark this answers is not in the document]
Comment (julien51) [reply unlock-app/src/components/Components.js]: Dammit. this is really annoying.

**Comment by julien51** [comment]
Heyo! Maybe a quick rebase against master so the eslint stuff us not on that PR?

**Comment by benwerd** [comment]
Ugggh, how did that happen. Yes!

File: unlock-app/.eslintrc.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ module.exports = {
     'airbnb',
     'eslint:recommended',
     'plugin:react/recommended',
+    'prettier',
```
Comment (julien51) [inline unlock-app/.eslintrc.js]: Shouldn't these changes be in another PR?

File: unlock-app/package.json [hunk 3]
```diff
@@ -75,7 +75,8 @@
     "lint": "./node_modules/eslint/bin/eslint.js .",
     "reformat": "npx prettier-eslint --write \"src/**/*.js\"",
     "storybook": "start-storybook -p 9001 -c .storybook -s src",
-    "svg-2-components": "./node_modules/@svgr/cli/bin/svgr --title-prop --no-dimensions -d src/components/interface/svg/ src/static/images/svg/"
+    "svg-2-components": "./node_modules/@svgr/cli/bin/svgr --title-prop --no-dimensions -d src/components/interface/svg/ src/static/images/svg/",
```
Comment (julien51) [inline unlock-app/package.json]: same here? You've done that in another PR, right?

**Comment by benwerd** [comment]
Rebased - PTAL when you can. Thanks!

**Comment by benwerd** [comment]
Really angry at myself for this PR. Somehow it missed a commit which busted the layout. I’ve got it working up to this point again, but I’m going to hang on until the whole responsive interface is in place.
